src/Backend/Modules/Blog/Installer/Installer.php

```diff
@@ -144,18 +145,18 @@ public function install()
         $this->setNavigation($navigationModulesId, 'Blog', 'blog/settings');
 
         // add extra's
-        $blogId = $this->insertExtra('Blog', 'block', 'Blog', null, null, 'N', 1000);
-        $this->insertExtra('Blog', 'widget', 'RecentComments', 'RecentComments', null, 'N', 1001);
-        $this->insertExtra('Blog', 'widget', 'Categories', 'Categories', null, 'N', 1002);
-        $this->insertExtra('Blog', 'widget', 'Archive', 'Archive', null, 'N', 1003);
-        $this->insertExtra('Blog', 'widget', 'RecentArticlesFull', 'RecentArticlesFull', null, 'N', 1004);
-        $this->insertExtra('Blog', 'widget', 'RecentArticlesList', 'RecentArticlesList', null, 'N', 1005);
+        $blogId = $this->insertExtra('Blog', ExtraType::block(), 'Blog', null, null, 'N', 1000);
+        $this->insertExtra('Blog', ExtraType::widget(), 'RecentComments', 'RecentComments', null, 'N', 1001);
+        $this->insertExtra('Blog', ExtraType::widget(), 'Categories', 'Categories', null, 'N', 1002);
+        $this->insertExtra('Blog', ExtraType::widget(), 'Archive', 'Archive', null, 'N', 1003);
+        $this->insertExtra('Blog', ExtraType::widget(), 'RecentArticlesFull', 'RecentArticlesFull', null, 'N', 1004);
+        $this->insertExtra('Blog', ExtraType::widget(), 'RecentArticlesList', 'RecentArticlesList', null, 'N', 1005);
 
         // get search extra id
         $searchId = (int) $this->getDB()->getVar(
             'SELECT id FROM modules_extras
              WHERE module = ? AND type = ? AND action = ?',
-            array('Search', 'widget', 'Form')
+            array('Search', ExtraType::WIDGET, 'Form')
```

Comment:
Because the query parameter should be a string. I felt it would be a bit silly to create an object from a string just to extract the string from it again. At first I used `(string) ExtraType::widget()` but it just seemed wrong, even though the `(string)` casting might not even be necessary